File: official/docs/java/addresses/VerifyExistingAddress.java
import com.easypost.EasyPost;
import java.util.HashMap;

public class VerifyExistingAddress {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        HashMap<String, Object> addressParams = new HashMap<String, Object>();

        addressParams.put("street1", "417 Montgomery Street");
        addressParams.put("city", "SF");
        addressParams.put("state", "CA");
        addressParams.put("zip", "94104");
        addressParams.put("country", "US");
        addressParams.put("company", "EasyPost");
        addressParams.put("phone", "415-123-4567");
        addressParams.put("verify_strict", true);

        Address address = Address.create(addressParams);

        Address verifiedAddress = address.verify();

        System.out.println(verifiedAddress);
    }
}

File: official/docs/php/addresses/VerifyExistingAddress.php
<?php

\EasyPost\EasyPost::setApiKey("EASYPOST_API_KEY");

$addressParams = [
    "street1" => "417 montgomery streat",
    "street2" => "FL 5",
    "city"    => "San Francisco",
    "state"   => "CA",
    "zip"     => "94104",
    "country" => "US",
    "company" => "EasyPost",
    "phone"   => "415-123-4567"
];

$address = \EasyPost\Address::create($addressParams);

$address->verify();

echo $addressses;
